src/main/java/eu/dissco/core/translator/configuration/DateSerializer.java

```diff
@@ -0,0 +1,29 @@
+package eu.dissco.core.translator.configuration;
+
+import static eu.dissco.core.translator.configuration.ApplicationConfiguration.DATE_STRING;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import java.io.IOException;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class DateSerializer extends JsonSerializer<Date> {
+
+  private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_STRING).withZone(
+      ZoneOffset.UTC);
```

Comment:
Is this a circular dependency? Not sure what the order of operations is here
- ApplicationConfiguration declares DATE_STRING
- ApplicatioConfiguration creates a bean of ObjectMapper, which contains a DateSerializer, which uses DATE_STRING

Is this possible because DATE_STRING is static? Is it safer to maybe declare DATE_STRING separately, maybe in a properties file?

Reply:
This is no different from the other serializers for instant. The DATE_STRING is static so not tied to an instance. It then starts creating all the beans and this bean can just use the static variable. I don't really see an issue here. Rather not make it a property as this should never change.